.github/workflows/publish-release.yml

```diff
@@ -6,7 +6,11 @@ on:
       - 'v*'
 
 jobs:
+  run-scala-ci-release:
+    uses: ./.github/workflows/standard-workflow.yml
```

Comment:
Out of curiosity... isn't it possible to merge this workflow file into what's now `.github/workflows/standard-workflow.yml`, this removing the need for `.github/workflows/standard-workflow.yml`? 🤔

Reply:
In other words... is it possible to define `on` on a per-`job` (and not per-workflow) level?

Reply:
> In other words... is it possible to define `on` on a per-`job` (and not per-workflow) level?

I think that it is not possible - see https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#on - `on` works on workflow-level.
However, I can try to get all the logic in one `ci.yml` file - and trigger it for:
- pushing to main
- pull request
- pushing tag
And just add one more `if` to the `publish-github-release` job to start only if the source event was a push of a git tag - **if it is possible**

Reply:
Not insisting, ignore if not easily doable

Reply:
But in my opinion the solution from PR - 3 separate files - is more clear and I would prefer it that way (two separate workflows for different triggering events). Let me know what is your opinion on that @PawelLipski

Reply:
Okay! let's leave then